pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md

```diff
@@ -9,7 +9,7 @@ Catalog applications are any third-party or open source applications that appear
 
 ## Upgrade catalog applications
 
-Before upgrading your catalog applications, verify the current and supported versions of the application.
+Before upgrading your catalog applications, verify the current and supported versions of the application. While platform applications must all have the same set and version of platform applications deployed, catalog applications are individual, so you can deploy and upgrade them individually for each project.
```

Comment:
How about this instead?

```suggestion
Before upgrading your catalog applications, verify the current and supported versions of the application. Also, keep in mind the distinction between Platform applications and Catalog applications.   Platform applications are deployed and upgraded as a set for each cluster or workspace.  Catalog applications are deployed separately, so that you can deploy and upgrade them individually for each project.
```